package.json

```diff
@@ -57,6 +61,7 @@
   "devDependencies": {
     "@chakra-ui/cli": "1.7.0",
     "@testing-library/cypress": "8.0.2",
```

Comment:
The Next.js Jest testing guide says to install [^guide]:

- `jest` <- Installed
- `@testing-library/react` <- Not installed
- `@testing-library/jest-dom` <- Not installed

Do you want to setup the `@testing-libary` stuff later on?

[^guide]: https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler

Reply:
That's right, I'll setup React Testing Library in a future PR.

I'm just testing plain JS functions for now, but we need to transform TS tests to be run with Jest. Instead of configuring that manually, I used Next built-in config for Jest.

Reply:
> That's right, I'll setup React Testing Library in a future PR.

Cool!

> Instead of configuring that manually, I used Next built-in config for Jest.

Yeah better to use built-in config than cobble something together yourself.